var/spack/repos/builtin/packages/steps/package.py

```diff
@@ -15,19 +17,25 @@ class Steps(CMakePackage):
     maintainers("tristan0x")
 
     license("GPL-3.0-or-later")
+    submodules = True
 
-    version("develop", branch="master", submodules=True)
-    version("4.1.1", submodules=True)
+    version("develop", branch="master")
+    version("5.0.1")
+    version("4.1.1")
```

Comment:
Can we pin the commit sha with `commit=`?

Reply:
sure @alalazo 
I added the SHA in c495bba5b244ae2344a0737403ad56b94197fa97